src/routes/safes/entities/safe-info.entity.ts

```diff
@@ -20,11 +20,11 @@ export class SafeState {
   readonly owners: AddressInfo[];
   @ApiProperty()
   readonly implementation: AddressInfo;
-  @ApiPropertyOptional()
+  @ApiPropertyOptional({ type: AddressInfo, nullable: true })
   readonly modules: AddressInfo[] | null;
```

Comment:
The type of this one should be an `Array<AddressInfo>`

Reply:
True 🤦🏻‍♂️
Fixed in [b25cf00](https://github.com/safe-global/safe-client-gateway/pull/765/commits/b25cf0019e537d984a8e1bdabaa4b0144d362acb), thanks!